includes/elements/container.php

```diff
@@ -443,6 +439,24 @@ protected function register_container_layout_controls() {
 			]
 		);
 
+		$this->add_responsive_control(
+			'max_height',
+			[
+				'label' => esc_html__( 'Max Height', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', 'vh', 'custom' ],
```

Comment:
```suggestion
				'size_units' => [ 'px', 'vh', 'vw', 'custom' ],
```

Reply:
I've noticed that sections use three unit types (`px`, `vh`, `vw`) while containers use two (`px`, `vh`). Let's fix that too.

Reply:
Is it really relevant to add the vw for the height, it seems to me that this unit is only suitable for the width?

Reply:
I don't mind having it there. If it's not there there will be one user that will ask for it :smile:

Reply:
Obviously, this is not disturbing in itself, other builders have all the units everywhere to offer something complete, because the code structure is different. Elementor wants to offer more units, just as the base code requires units to be added to containers and elements on a case by case basis for each of them, so we might as well think about this detail.